main/templates/main/state.html

```diff
@@ -52,25 +52,25 @@ <h5 class="mt-3">Communities of Interest</h5>
   </div>
   <div class="card card-signin my-3">
     <div class="card-body">
-      <h2 class="font-weight-light">Data Collection Campaigns in {{ state_obj.name }}</h2>
-      <p>Organizations in {{ state_obj.name }} are running campaigns to collect community information so your community's interests can be protected. You can use Representable to submit your community's information directly to these organizations.</p>
-      {% if campaigns %}
+      <h2 class="font-weight-light">Data Collection Drives in {{ state_obj.name }}</h2>
```

Comment:
Small note: I think we should rename this from Data Collection Drives to Community Mapping Drives

Reply:
ooh I did not know this was here, I can change that